src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/ReadmePreviewViewModel.cs

```diff
@@ -60,6 +60,7 @@ public async Task SetPackageMetadataAsync(DetailedPackageMetadata packageMetadat
         {
             if (packageMetadata != null && (!string.Equals(packageMetadata.Id, _packageMetadata?.Id) || packageMetadata.Version != _packageMetadata?.Version))
             {
+                ReadmeMarkdown = Resources.Text_Loading;
```

Comment:
Additionally, if any code needs to check on whether the ViewModel is still in the loading state, it's preferable for it to check the IsLoading property instead of checking if the ReadmeMarkdown == Resources.Text_Loading in the appropriate language.